### PR TITLE
Claroty-CTD-Modeling-Rules-CIAC-7022

### PR DESCRIPTION
**Pull request by cweltPA**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-7022](https://jira-dc.paloaltonetworks.com/browse/CIAC-7022)

## Description
This PR contains XSIAM Modeling Rules content for the Claroty CTD (Continuous Threat Detection) pack. 

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by eepstain — APPROVED** [review]

File: Packs/Claroty/README.md [hunk 1]
```diff
@@ -7,5 +10,54 @@ Supporting the broadest list of OT protocols in the industry and multiple asset
...
+## Configuration on Server Side
+Follow these steps to configure Claroty CTD to forward Syslog messages to Cortex XSIAM.
+ 
+1. Login to your account on the Claroty CTD web management console. 
+2. Go to **Configuration** and navigate to **Log Settings** &rarr; **Syslog**.
+3. Click **+ Add** to add a new syslog configuration. 
+4. Clear the **Local** checkbox and fill in the following settings: 
+   | Parameter          | Value   
+   | :---               | :---        
+   | `Message Contents` | Select the log type you wish to forward to Cortex XSIAM.
```
Comment (ShirleyDenkberg) [inline Packs/Claroty/README.md:31]:
```suggestion
   | `Message Contents` | Select the log type to forward to Cortex XSIAM.

```

File: Packs/Claroty/README.md [hunk 2]
```diff
@@ -7,5 +10,54 @@ Supporting the broadest list of OT protocols in the industry and multiple asset
...
+   | `Message Format`   | Select **CEF**.
+   | `Server`           | Enter the IP Address of the target Cortex XSIAM Broker VM syslog server.
```
Comment (ShirleyDenkberg) [inline Packs/Claroty/README.md:33]:
```suggestion
   | `Server`           | Enter the IP address of the target Cortex XSIAM Broker VM syslog server.

```

File: Packs/Claroty/README.md [hunk 3]
```diff
@@ -7,5 +10,54 @@ Supporting the broadest list of OT protocols in the industry and multiple asset
...
+   | `Port`             | Enter the port number which the target Cortex XSIAM Broker VM syslog server would be listening on for receiving syslog messages from Claroty CTD.
+   | `Protocol`         | Select the requested forwarding transport protocol (*UDP*, *TCP* or *TLS*). 
+5. Click **Save**.
+   
+### Remark
+Since the syslog forwarding configuration is set for each log type individually, 
+repeat the steps above for each log type (*Alerts*, *Events*, etc.) that you wish to monitor on Cortex XSIAM.
```
Comment (ShirleyDenkberg) [inline Packs/Claroty/README.md:40]:
```suggestion
repeat the steps above for each log type (*Alerts*, *Events*, etc.) to monitor on Cortex XSIAM.

```

**Comment by ShirleyDenkberg** [comment]
@eepstain Doc review completed.

**Review by eepstain — APPROVED** [review]

**Comment by cweltPA** [comment]
@ShirleyDenkberg 
I added a small disclaimer on top of the XSIAM section on the README, 
if you can review it once again

**Comment by ShirleyDenkberg** [comment]
@cweltPA Looks OK.

**Pull request merged**